webextension/background/takeshot.js

```diff
@@ -35,9 +37,10 @@ window.takeshot = (function () {
       clipboard.copy(shot.viewUrl);
       chrome.notifications.create(id, {
         type: "basic",
-        iconUrl: "img/clipboard-32.png",
+        iconUrl: "../icons/clipboard-32.png",
         title: "Link Copied",
-        message: "The link to your shot has been copied to the clipboard"
+        message: "The link to your shot has been copied to the clipboard. Press "
+        + pasteSymbol + "-V to paste."
```

Comment:
We also use ES6 here, so you can do:

    `The link to your shot has been copied to the clipboard. Press ${pasteSymbol}-V to paste.`

but this is fine too